src/celeritas/em/msc/UrbanMscStepLimit.hh

```diff
@@ -81,11 +81,14 @@ class UrbanMscStepLimit
     // Urban MSC range properties
     MscRange msc_range_;
 
-    bool on_boundary_{};
```

Comment:
How is this condition (the first step of the tracking or the first step in the new volume) treated now?

Reply:
It's evaulated in the constructor rather than at `operator()` time, so we don't need the member data any more.